org/mozilla/jss/pkcs11/PKCS11Constants.java

```diff
@@ -0,0 +1,8623 @@
+/**
+ * PKCS11Constants.java - part of the JSS project
+ * Copyright )C) 2018 Red Hat, Inc.
+ *
+ * This file is automatically generated (tools/build_pkcs11_constants.py)
+ * from the contents of NSS's pkcs11t.h and pkcs11n.h headers. This
+ * replaces the sun.security.pkcs11.wrapper.PKCS11Constants interface
+ * which was removed from public visibility in JDK 9 due to the
+ * introduction of modules.
+ *
+ * For more information, please see the documentation under
+ * jss/docs/pkcs11_constants.md.
+ *
+ * Below are the copyright statements for the sourced files:
+**/
+/* Copyright statements from /usr/include/nss3/pkcs11t.h */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* License to copy and use this software is granted provided that it is
+ * identified as "RSA Security Inc. PKCS #11 Cryptographic Token Interface
+ * (Cryptoki)" in all material mentioning or referencing this software.
+
+ * License is also granted to make and use derivative works provided that
+ * such works are identified as "derived from the RSA Security Inc. PKCS #11
+ * Cryptographic Token Interface (Cryptoki)" in all material mentioning or
+ * referencing the derived work.
+
+ * RSA Security Inc. makes no representations concerning either the
+ * merchantability of this software or the suitability of this software for
+ * any particular purpose. It is provided "as is" without express or implied
+ * warranty of any kind.
+ */
+
+/* Copyright statements from /usr/include/nss3/pkcs11n.h */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.jss.pkcs11;
+
+public interface PKCS11Constants {
+    /**
+     *
+     * Content automatically generated; see NSS documentation for more information.
+     *
+     * Source file: /usr/include/nss3/pkcs11t.h
+     * Line number: 103
+     * Line: #define CKN_SURRENDER 0
+     * Parsed name: CKN_SURRENDER
+     * Parsed value: 0
+     *
+    **/
```

Comment:
Generally the comment block should end with just */ instead of **/ and the * should align with the previous line. See also: https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#examples

This type of comment block will be used by Javadoc tool to generate documentation content.